src/ZkFold/Symbolic/Cardano/Contracts/SymbolicWrapper.hs

```diff
@@ -0,0 +1,98 @@
+{-# LANGUAGE TypeOperators #-}
+module ZkFold.Symbolic.Cardano.Contracts.SymbolicWrapper where
```

Comment:
I would rename this module like this: `ZkFold.Symbolic.Cardano.ContractWrapper`.